### PR TITLE
Implement new MemberUpdateService for new add/remove Wrapper Paradigm.

### PR DESCRIPTION
**Pull request by zakgilbert**
Update wrapper command types to check for null params. Rename command test files to suffix with 'test' rather than prefix.

# Ticket Link

[GROUPINGS-1247]()https://uhawaii.atlassian.net/browse/GROUPINGS-1247

# Test Checklist

- [x] Unit Tests Passed:
- [x] Integration Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

**Comment by zakgilbert** [comment]
The main thing to look at here is service/UpdateMemberService.

**Review by mhodgesatuh — COMMENTED** [review]
This will be a tricky one given how many files are changing at once.

**Comment by mhodgesatuh** [comment]
Codacy is complaining that this following is too comlex:
    public void checkIfOwnerOrAdminUserTest() {

**Comment by zakgilbert** [comment]
> This will be a tricky one given how many files are changing at once.

Lucky none of it affects the UI, all has to do with new wrapper paradigm.

**Comment by zakgilbert** [comment]
> Codacy is complaining that this following is too comlex: public void checkIfOwnerOrAdminUserTest() {

I gotta research what that error means, but at just a glance I wonder how a test function with loops could throw an error as such.

**Comment by zakgilbert** [comment]
Fixed the codacy NPath problem.

**Comment by mhodgesatuh** [comment]
MemberService has a commented out method that likely should be removed: GroupingHasMemberResult.

**Comment by zakgilbert** [comment]
a furthur explanation pertaining to the size of this PR can be seen in the ticket 
https://uhawaii.atlassian.net/browse/GROUPINGS-1247

File: src/main/java/edu/hawaii/its/api/service/MemberService.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        return isGroupMember(includePath, uhIdentifier);
+    }
+
+    public boolean isExcludeMember(String groupingPath, String uhIdentifier) {
+        return isGroupMember(groupingPath + ":exclude", uhIdentifier);
+    }
+
+    public boolean isOwner(String groupingPath, String uhIdentifier) {
+        return isGroupMember(groupingPath + ":owners", uhIdentifier);
+    }
```
Comment (JorWo) [inline src/main/java/edu/hawaii/its/api/service/MemberService.java:48]: Your isExcludeMember and isOwner functions may be missing the `throw new InvalidGroupPathException`. I am working on a ticket where I am planning on using this service; could you add a isBasisMember function? Thanks!

Comment (zakgilbert) [reply src/main/java/edu/hawaii/its/api/service/MemberService.java:48]: I see, I would stick with MemberAttributeService in your implementation for now, these services I implemented are not ready to be used in working code.

Comment (zakgilbert) [reply src/main/java/edu/hawaii/its/api/service/MemberService.java:48]: But thanks for bringing this up, "isIncludeMember()" is not correct.

**Comment by JorWo** [comment]
There are some leftover rebase comments in GroupingsMailService.

**Comment by zakgilbert** [comment]
This PR is required for [GROUPINGS-1330-Reset Group not working for large groupings](https://uhawaii.atlassian.net/browse/GROUPINGS-1330) to work.

**Review by mhodgesatuh — COMMENTED** [review]
Appears to be knocking out the "helper" comments recently added.

**Comment by zakgilbert** [comment]
Helper comments have been fixed.

**Review by mhodgesatuh — APPROVED** [review]
I don't see any issues.

**Pull request merged**